Resources/public/js/views/services/plugins/extensions/ez-draftserversidevalidation.js

```diff
@@ -32,22 +32,26 @@ YUI.add('ez-draftserversidevalidation', function (Y) {
          */
         _parseServerFieldsErrors: function (response, serverSideErrorCallback) {
             var serverSideFieldsError = [],
-                error = response.document.ErrorMessage,
-                fieldsError = error.errorDetails ?  error.errorDetails.fields : [];
+                error,
+                fieldsError;
 
             if (serverSideErrorCallback) {
-                fieldsError.forEach(function (field) {
-                    field.errors.forEach(function (error) {
-                        var serverSideFieldErrorDetails;
+                if ( response ) {
```

Comment:
how is that possible ? `response` is documented as required actually calling this method without a response is odd since the point of this method is to parse the response.